pycbc/conversions.py

```diff
@@ -1545,3 +1549,82 @@ def nltides_gw_phase_diff_isco(f_low, f0, amplitude, n, m1, m2):
            'nltides_gw_phase_diff_isco', 'spin_from_pulsar_freq',
            'freqlmn_from_other_lmn', 'taulmn_from_other_lmn'
           ]
+
+##############################################################################
+# NS-BH merger remnant mass [Foucart, Hinderer, Nissanke PRD 98, 081501(R)   #
+# (2018)].                                                                   #
+#                                                                            #
+# * THIS ASSUMES THE NS SPIN IS 0 (the user is warned about this).           #
+# * Physical parameters passed to remnant_mass (see sanity checks below)     #
+#   must not be unphysical.                                                  #
+# * Tilted BH spin cases are handled by using rISSO(chi,incl) where rISCO    #
+#   appears in the formula. This approximation was suggested in Stone, Loeb, #
+#   Berger, PRD 87, 084053 (2013) for the previous NS-BH remnant mass fit of #
+#   Foucart, PRD 86, 124007 (2012).                                          #
+##############################################################################
+def remnant_mass(eta, ns_g_mass, ns_sequence, chi, incl):
+    """
+    Function that determines the remnant disk mass of
+    an NS-BH system using the fit to numerical-relativity
+    results discussed in Foucart, Hinderer, Nissanke PRD 98, 081501(R) (2018).
+
+    Parameters
+    -----------
+    eta: float
+        the symmetric mass ratio of the binary
+    ns_g_mass: float
+        NS gravitational mass (in solar masses)
+    ns_sequence: 3D-array
+        contains the sequence data in the form NS gravitational
+         mass (in solar masses), NS baryonic mass (in solar
+         masses), NS compactness (dimensionless)
+    chi: float
+        the BH dimensionless spin parameter
+    incl: float
+        the inclination angle between the BH spin and the orbital
+        angular momentum in radians
+
+    Returns
+    ----------
+    remnant_mass: float
+        The remnant mass in solar masses
+    """
+
+    # Sanity checks on eta and chi
+    if not (eta>0. and eta<=0.25 and abs(chi)<=1 and ns_g_mass>0):
+        err_msg = "The BH spin magnitude must be <= 1,"
+        err_msg += "eta must be between 0 and 0.25,"
+        err_msg += "and the NS mass must be positive."
+        logging.error(err_msg)
+        info_msg = "The function remnant_mass was launched with"
+        info_msg += "ns_mass={0}, eta={1}, chi={2},"
+        info_msg += "inclination={3}\n'.format(ns_g_mass, eta, chi, incl)"
+        logging.info(info_msg)
+        sys.exit(1)
```

Comment:
Please remove the `logging.error()` and `sys.exit()` calls, change the exception to a `ValueError`, and show the full error message in the exception instead.

Reply:
Then you can also remove the sys import above.